aflr/helper_classes.py

```diff
@@ -0,0 +1,63 @@
+from .api_request import APIRequest
+
+
+class ListableResource(APIRequest):
```

Comment:
Is this spelt right? Is listable a word?

Reply:
it is a word, I checked.
Also, this nomenclature was inspired in stripe's one

Reply:
it doesnt have to be a word tho, end users wont see it